flink-runtime-web/web-dashboard/src/app/interfaces/job-exception.ts

```diff
@@ -44,7 +44,8 @@ export interface ExceptionInfo {
   stacktrace: string;
   timestamp: number;
   taskName: string;
-  location: string;
+  endpoint: string;
+  taskManagerId: string;
```

Comment:
Why should we add the 'taskManagerId' as a hotfix? It doesn't look like a bug or a refactor.

Reply:
I added it as hotfix because the field is used in "flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.html" L85 but not declared in the `ExecptionInfo` interface, and Idea Inteiij throws an error: "Unresolved variable taskManagerId". However, I'm not familiar with TS and I have no idea why the code can still run no matter the field is declared in the interface or not. @yangjunhan Could you help take a look here? Many thanks in advance.

Reply:
Get, It may be missing from the previous commit, LGTM.